Source/Core/VideoBackends/OGL/OGLGfx.cpp

```diff
@@ -430,7 +436,31 @@ void OGLGfx::PresentBackbuffer()
   }
 
   // Swap the back and front buffers, presenting the image.
-  m_main_gl_context->Swap();
+  if (Common::VR::IsEnabled())
+  {
+    if (!m_vr_initialized)
+    {
+      Common::VR::Start(true);
```

Comment:
Why isn't this in the constructor when the backend is initialized?

Reply:
It has to be called from the thread with GLContext.